### PR TITLE
Schema generation

### PR DESCRIPTION
**Pull request by MarekBodingerBA**
Schemas need to be updated on STG while merging. Commit messages should be self-explanatory.

### CODE REVIEW

File: next/frontend/utils/form.ts [hunk 1]
```diff
@@ -71,12 +57,27 @@ export const getAjvFormKeywords = (
...
+  return t1 - t2
+}
+
+export const ajvFormats: CurrentOptions['formats'] = {
   zip: /\b\d{5}\b/,
   // TODO: Remove, but this is needed for form to compile
   ciselnik: () => true,
-  localTime: () => true,
+  // https://blog.kevinchisholm.com/javascript/javascript-e164-phone-number-validation/
+  'phone-number': /^\+?[1-9]\d{1,14}$/,
```
Comment (mpinter) [inline next/frontend/utils/form.ts:75]: if we're doing it this way (=regex way), then let's remove the libphonenumber validation elsewhere which passes a wider range of numbers (I agree that we don't need to allow those). May need a very slight update to error translations ('number without spaces').

File: next/schema-generator/cli.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  )
+
+  fs.writeFileSync(
+    'C:\\Projects\\konto.bratislava.sk\\next\\backend\\forms\\test\\schema.json',
+    JSON.stringify(definition.schema, null, 2),
+  )
+  fs.writeFileSync(
+    'C:\\Projects\\konto.bratislava.sk\\next\\backend\\forms\\test\\uiSchema.json',
+    JSON.stringify(definition.uiSchema, null, 2),
+  )
```
Comment (mpinter) [inline next/schema-generator/cli.ts:40]:
```suggestion
```

I guess this is a forgotten test code

File: next/schema-generator/generator/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  }))
+
+export const createCamelCaseOptions = (options: string[]) =>
+  options.map((option, index) => ({
+    value: camelCase(option),
+    title: option,
+    isDefault: index === 0 ? true : undefined,
+  }))
+
+export const createCamelCaseOptionsV2 = <Option extends { title: string }>(options: Option[]) =>
```
Comment (mpinter) [inline next/schema-generator/generator/helpers.ts:84]: extra comment might be worth it

**Review by mpinter — APPROVED** [review]
Will test once I get the xml stuff running (also possibly include it in the schema-generator).

Looking good, check comments & lgtm 🚢

File: next/schema-generator/generator/helpers.ts (hunk 3 above)
Comment (MarekBodingerBA) [reply next/schema-generator/generator/helpers.ts]: Added comment.

File: next/frontend/utils/form.ts (hunk 1 above)
Comment (MarekBodingerBA) [reply next/frontend/utils/form.ts:75]: Created issue
https://github.com/bratislava/konto.bratislava.sk/issues/330

**Review by mpinter — APPROVED** [review]

**Pull request merged**